### PR TITLE
data(population-education): Add Wittgenstein's dataset

### PR DESCRIPTION
**Pull request by lucasrodes**
Add Population size by education level dataset. This should update our existing dataset [Global Projection, Medium SSP2 - IIASA (2016)](https://owid.cloud/admin/datasets/225).

The processing is straightforward; I barely changed the template from `walkthrough`. The only changes are in the following:

- `snapshots/demography/2023-04-07/population_education_wittgenstein.py`: Read the file locally and added some instructions to download the file.
- `data/meadow/demography/2023-04-07/population_education_wittgenstein.py`: Read the CSV from snapshot, skipping the first eight rows (they are not well-formatted). Sanity check: Ensure that row 9 (the table header) is as expected.
- `data/garden/demography/2023-04-07/population_education_wittgenstein.py`: Renamed column, set indexes.

### CODE REVIEW

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# (Inherited from meadow, remove if not different.)
+all_sources:
+  - sources: &sources
+      name: Wittgenstein Centre (2018)
+      published_by: Wittgenstein Centre for Demography and Global Human Capital (WIC) Wittgenstein Centre Data Explorer. Version 2.0 2018
+      url: http://dataexplorer.wittgensteincentre.org/wcde-v2/
+      date_accessed: 2023-04-07
+      publication_year: 2018
+      # description: Source description.
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:9]: If you haven't changed anything, you could simply remove this and import the source directly from snapshot and meadow.

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# (Inherited from meadow, remove if not different.)
+dataset:
+  sources:
+    - *sources
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:14]: Idem here.

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  population_education_wittgenstein:
+    # (Inherited from meadow, remove if not different.)
+    variables:
+      population:
+        title: Population
+        unit: persons
+        description: |
+          Population projections by levels of educational attainment.
+
+          • Under 15: Population below 15 years old
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:26]: Missing period at the end of sentence.

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+          • No Education: Highest level of education attained: No level and Grade 1 of ISCED 1 not completed.
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:27]: Maybe explain acronym ISCED the first time you use it.

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+          • Incomplete Primary: Highest level of education attained: Incomplete ISCED 1
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:28]: Missing period at the end of sentence. This happens to all sentences below.

File: etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+          • Primary: Highest level of education attained: Completed ISCED 1 and incomplete ISCED 2
+          • Lower Secondary: Highest level of education attained: Completed ISCED 2 and incomplete ISCED 3
+          • Upper Secondary: Highest level of education attained: Completed ISCED 3 and incomplete ISCED 4 5 or 6
+          • Post Secondary: Highest level of education attained: Completed ISCED 4 5 6 7 or 8
+          • Short Post Secondary: Highest level of education attained: Completed ISCED 4 or 5
+          • Bachelor: Highest level of education attained: Completed ISCED 6 and incomplete ISCED 7
+          • Master and higher: Highest level of education attained: Completed ISCED 7 or 8
+        sources:
```
Comment (pabloarosado) [inline etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml:36]: If sources are identical to dataset sources, then I think you don't need to specify them here.

**Review by pabloarosado — APPROVED** [review]
All good, just minor suggestions.

File: snapshots/demography/2023-04-07/population_education_wittgenstein.csv.dvc [hunk 7]
```diff
@@ -0,0 +1,31 @@
+meta:
+  namespace: demography
+  short_name: population_education_wittgenstein
+  name: Population size by education level (Wittgenstein Centre, 2018)
+  version: 2023-04-07
+  publication_year: 1974
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply snapshots/demography/2023-04-07/population_education_wittgenstein.csv.dvc:6]: typo, thanks!

**Pull request merged**